00:00:00.001200 main     VirtualBox COM Server 6.1.16 r140961 win.amd64 (Oct 16 2020 16:33:17) release log
00:00:00.001203 main     Log opened 2020-11-23T15:30:59.846835800Z
00:00:00.001203 main     Build Type: release
00:00:00.001206 main     OS Product: Windows 10
00:00:00.001207 main     OS Release: 10.0.18363
00:00:00.001208 main     OS Service Pack: 
00:00:00.014551 main     DMI Product Name: 20L6S9U200
00:00:00.021005 main     DMI Product Version: ThinkPad T480
00:00:00.021014 main     Firmware type: BIOS
00:00:00.021018 main     Host RAM: 32426MB (31.6GB) total, 23607MB (23.0GB) available
00:00:00.021023 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.021024 main     Process ID: 11976
00:00:00.021024 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.079195          VirtualBox: object creation starts
00:00:00.080425          Home directory: 'F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox'
00:00:00.080515          Installed Drivers:
00:00:00.084203            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.16.40961)
00:00:00.084568            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.16.40961)
00:00:00.084807            C:\WINDOWS\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.16.40961)
00:00:00.085223            C:\WINDOWS\System32\drivers\VBoxNetAdp6.sys (Version: 6.1.16.40961)
00:00:00.087993          Loading settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.089194          usbLibGetDevices: Starting USB device enumeration
00:00:00.090520          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.090561          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 7
00:00:00.090595          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:00:00.090628          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 9
00:00:00.091520          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 15
00:00:00.091730          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.092759          usbLibGetDevices: Found 10 USB devices, 0 captured
00:00:00.092831          USB: Unknown USB device detected (idVendor: 0x06cb, idProduct: 0x009a)
00:00:00.092844          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x0316)
00:00:00.092857          USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x56a6)
00:00:00.092874          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0a2b)
00:00:00.092888          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0ac9)
00:00:00.170675          HostDnsMonitor: initializing
00:00:00.183244          HostDnsMonitor: updating information
00:00:00.183271          HostDnsMonitor: old information
00:00:00.183272            no server entries
00:00:00.183274            no domain set
00:00:00.183274            no search string entries
00:00:00.183277          HostDnsMonitor: new information
00:00:00.183278            server 1: 127.0.0.1
00:00:00.183279            domain: ni.corp.natinst.com
00:00:00.183280            no search string entries
00:00:00.299912          VD: VDInit finished with VINF_SUCCESS
00:00:00.301101          Loading settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:00:00.302730          Loading settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\Debug\Debug.vbox" with version "1.14-windows"
00:00:00.313857          OCI: Local config file 'F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\oci_config' does not exist
00:00:00.313962          OCI: Original config file 'C:\Users\inthirua\.oci\config' does not exist
00:00:00.313964          OCI: Reading profiles finished with status ERROR_SUCCESS
00:00:00.313972          ExtPack: Created cloud provider 'OCI' (hrc=ERROR_SUCCESS)
00:00:00.313995          VirtualBox: object created
00:00:00.810667          Launched VM: 72537344 pid: 14120 (0x3728) frontend: GUI/Qt name: LinuxLive
00:00:04.476227          Load [C:\Program Files\Oracle\VirtualBox\ExtensionPacks\Oracle_VM_VirtualBox_Extension_Pack\win.amd64\VBoxHostWebcam.DLL] vrc=VINF_SUCCESS
00:00:04.709013          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:01:31.129350          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:01:31.289962          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:01:37.384834 main     VirtualBox: object deletion starts
00:01:38.087189 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\HardDisks/LinuxLive.vmdk' cannot be closed because it is still attached to 2 virtual machines}, preserve=false aResultDetail=0
00:01:38.093719 main     HostDnsMonitor: shutting down ...
00:01:38.099061 main     HostDnsMonitor: shut down
00:01:38.106880 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
00:01:38.112132 main     VirtualBox: object deleted
